tests/baselines/reference/user/webpack.log

```diff
@@ -1,37 +1,842 @@
 Exit Code: 1
 Standard output:
+lib/AmdTemplatePlugin.js(42,16): error TS2339: Property 'hooks' does not exist on type 'Stats'.
```

Comment:
this is pretty bad! I'm bisecting right now to figure out what broke this. Afterwards I'll produce a small repro.